### ISSUE TITLE
On Nyxt's PR

### ISSUE THREAD

**Issue by aadcg** [atlas-engineer/nyxt#2166]
Nyxt defines itself to be a web browser that is "infinitely extensible".

I find that "infinitely" is redundant, since the concept of extensibility already makes clear that users can get more than what's available out of the box. However, there's a dimension of Nyxt that could be emphasized. 

Nyxt isn't a program that you can "simply" change the source code and generate a derived work. Rather, it makes an honest effort to "babysit" the users and let every knob be easily tweakable. 

Therefore, I suggest highlighting "extensibility" and "introspectability" that Nyxt provides out of the box, even for the less technical users.

**Comment by Ambrevar** [comment]
Fair enough! :)  Do you have a specific suggestion in mind?

**Comment by Ambrevar** [comment]
I agree with all your points! :)

@jmercouris ?

**Comment by jmercouris** [comment]
I know that you agree Pierre :-D

I am however of a different mentality. If we don't take ourselves seriously, if we don't think our product is good. Who does? Why should they listen to us? I mean this is largely a cultural thing. As a generalization: Northern Europeans tend to understate things, and Americans tend to overstate things.

Perhaps it is hubris, perhaps the gods shall punish us. I don't know. I would be OK with removing the word "Power browser" and calling it `the hacker's browser`. This still encompasses all things I wish to say. We could also consider `the hackable browser`.

**Comment by Ambrevar** [comment]
I don't think it's about not taking ourselves seriously, quite the opposite: bragging over the top translates into insecurity in my opinion.  If we want to be serious, gotta be real.

Also I find it less self-entitled to describe the product and not the audience.  Let the audience decide for themselves if they want to consider themselves "power users", "hackers", "tech enthusiasts" or what not.

I like the `hackable browser`.

**Comment by aadcg** [comment]
I agree with @Ambrevar. 

I'd be careful with any anything related to "hacking". We know what the "hacking culture" is, but most people (including our users) don't. Those who identify themselves with the culture won't feel less at home by the lack of "memes" and anecdotes.

The more "elite" or "niche" we try to be, the more we harm ourselves.

**Comment by aadcg** [comment]
Regarding taking ourselves seriously, I agree that we need to be decisive and resolute. In that regard,  we need to start sharing the word offline. I believe I'd be able to visit some universities, companies or meetups in Portugal to deliver a 1-hour presentation.

**Comment by lansingthomas** [comment]
Sorry for the long essay here. I have been thinking about Nyxt's value proposition... I provide a few specific questions in comments below to move the discussion forward.


## Tom's Principles / Specifications for Messaging
1. messaging for products ought to be very, very clear. [memorable, connected, value driven]
   - [ ] **memorable -** message must be so simple that customers can **remember** it and **explain** it to a friend in a few words.
   - [ ] **connected -** message should connect with things that customers already understand. (I think Nyxt's message has done well so far by being connected to Emacs, we can connect it to other things as we grow)
   - [ ] **value driven -** message MUST communicate our value (value proposition of the product, as well as our values as a group {devs + users})
2. It is important to make a declaration about our product.
- no product is for everyone, so making an assertion about our value does not really have a downside. (_are you a hacker? GREAT WELCOME TO NYXT!!!  ELSE; feel free to keep using other browsers, we have no expectations!_) 
- a lot of successful PR messaging follows a really simple formula -- `for those who believe (__), we see you. And we assert (___)` 
- so for nyxt it might be something like:
   > for those with a long list of online tasks, a long list of search results only gets you so far. We assert that computationally augmented browsing is far superior for modern Internet work, and Nyxt is a free and fantastic access point for that experience.

## Things I like for our message
#### I believe we should communicate these FOUR things in one simple message
1. Nyxt enables computationally augmented browsing. (_a new and exciting way to use the computer!_)
2. Nyxt creates a sense of ownership in digital environments
3. Nyxt blends other tools into a single tool {window managers, text editors, AI tools, ...}
4. Nyxt is all about the keyboard, and you should be as well.


#### `hackable browser` sounds okay to me. The concept of hacking in our messaging I like.
 - I notice that dictionary definitions of `hack` and `hacking` are all negative and I think that they have this wrong. For Nyxt's audience -- hacking is a good thing.  
- in my view; hacking communicates a better, faster, or smarter way to get something done. hacking is about getting exactly what you want from a complex environment.
- hacking as a concept is growing to encompass a wider audience, and the term is inclusive for our potential new users (life hacks, work hacks, homework hacks, computer hacks, health hacks), and has positive connotations both on university campuses and in developer-culture land. (this is our audience and they like hacking)
 - Neil Stephenson (Snowcrash et al.), metaverse, Hackernews, and hacker culture are all cool right now. (build for what is, not for what will be -- its good timing!)

  
 #### `knowledge extraction engine` sounds better to me.
- I have heard John pitch this and it seems like an effective way to converse about HOW Nyxt is helpful, for those who already align with our values.
- a `knowledge extraction engine` seems like it works out of the box. I don't feel the need to `hack` it before it works.  This is very good. 
- Engines are clean, precise, modern, complex yet useful. _Fuel in &rarr; power out_. Nyxt is like _data in &rarr; knowledge out._


## Things I do NOT like for our message
#### I believe we should NOT communicate these things in our simple message:
1. we should NOT give people the idea that they MUST hack anything to use Nyxt. (_instead: we are open source, with no expectations_)
2. we should NOT give people the idea that Nyxt is another browser. (_instead: we are something better than browsers as a concept_)

#### `power browser` i do not like
   - vagueness
   - presents Nyxt as a lot more stable than it is
   - sounds like power lifting, powertrain, ... it feels super NOT nerdy to me. (and we are nerdy!)
   - 

#### `the Internet on your terms` I do not like.
- very vague, almost unbelievably vague
- IF I have some specific customizations in mind, I probably found Nyxt by searching for a term like `window manager browser integration', or `in browser text editor` ---- I don't think that this user is going to be moved by `the internet on your terms` -- they are already taking matters into their own hands, they need a tool, not the idea of doing things their own way.

---

# Value / Affordances Users Might Earn By Adopting Nyxt

(I have bolded the few that I think are most important -- and should be part of our message)

### Social Impact Elements of Value
Paths to open-source values and self-transcendence
Aligns with value: Transparency
Emancipates from surveillance capitalism and tedious browsers
 
### Life Changing Elements of Value
**Enables computationally augmented browsing**
Stay in focused flow (minimized distraction)
Rewards with personalized custom fixes
Feels nostalgic for the digital brutalism aesthetic
Represents a nerdy status achievement
Enables micro-niche workflows
Entertains the Build Your Own Worlders
Embodies open source philosophy
Provides programmable access for code-builders to the web
 
### Emotional Elements of Value
Hope for complete understanding of digital world
Self-actualization as a computer professional / digital native / tech wizard
Motivates learning {web tool building, Lisp, emacs, …}
Motivates customization of digital spaces
Affiliates with super-mega geeks
**Creates a sense of ownership in digital environments**
Avoids worry/concern over future work
Sings freedom for users (nothing off limits)
 
### Functional Elements of Value
Saves time for online tasks
Simplifies online tasks
Classifies data/information {builds models of understanding}
**Integrates with other tools {window managers, text editors, AI tools, ...}**
Connects disparate browsing structures {history, annotations, tags, bookmarks, …}
**Sensorily appeals to keyboard enthusiasts**
Sensorily appeals to digital minimalists
Minimizes distractions
Avoids hassles online
Reduces effort for online tasks
Teaches and practices Common Lisp
Teaches and practices buffers
Provides some variety for computer enthusiasts
Prototypes customizations live (without compiling)
Provides quest-able environment to learn digital customizations
Provides longevity of effort with macros for dynamic code generation
Self-documents and introspects (glass box v black box)
Emulates beloved features from other browsers
Blocks advertisements
Searches natively with any search engine
Re-finds ongoing tasks
Re-surfaces ongoing tasks
Reminds ongoing tasks
navigates the web
extracts information from the Internet
Presents source-code for introspection
Presents source-code for modification
Presents source-code for customization

**Comment by lansingthomas** [comment]
Would you include anything else in this (hopefully comprehensive) declaration of value? {functional, emotional, life changing, social}?

**Comment by lansingthomas** [comment]
Which of these elements of value are the most important in your view? (we should try to narrow our focus)

**Comment by lansingthomas** [comment]
TLDR; take ourselves seriously, call it `knowledge extraction engine` until someone thinks of something better, perhaps agree on what we think the real value of Nyxt might be.

**Comment by lansingthomas** [comment]
@Ambrevar @jmercouris @aadcg @aartaka

**Comment by aadcg** [comment]
> I notice that dictionary definitions of hack and hacking are all negative and I think that they have this wrong. For Nyxt's audience -- hacking is a good thing.

Sadly, hacking is now perceived as "cracking". In short, hacking can be defined as playful cleverness (it does not always need to be "useful"). You may be interested in reading "Hackers: Heroes of the Computer Revolution" by Steven Levy. It's the best book I know that explores the hacker culture. 

> TLDR; take ourselves seriously, call it knowledge extraction engine until someone thinks of something better, perhaps agree on what we think the real value of Nyxt might be.


While I agree with many of the points that you're making, perhaps even "knowledge extraction engine" is a bit narrow. It gives the idea that you're a passive user, and all you do is extracting knowledge. But Nyxt equally allows users to synthesize new pieces of knowledge too! 

Also, while Nyxt is arguably better at extracting knowledge from the Internet than other web browsers, those too could be labelled as such. Think of the Web/Internet, a knowledge repository, as a chunk of marble. Web browsers are the tools to allows users to fetch the raw material and some limited level of sculpting. I think that our value (the problem we're solving) is as follows: serious sculptors use mainstream web browsers (there's no alternative), since it's their gateway to the chunks of marble. But then, they take it to their own ateliers where they can use their own set of tools. Our proposition is that you can have the atelier where you fetch marble. 

We should not forget that the Internet is much more than a knowledge repository too. It's used for bookkeeping, communication and so many other things!

**Comment by aartaka** [comment]
I agree with points of André, both about hacking as a mostly negative term and knowledge extraction as too narrow. That's why we kind of have these vague slogans like "Internet on your terms": we are Internet citizens, but also a more involved and productive kind of citizens in most of activities that we participate in, not only in knowledge extraction.

**Comment by jmercouris** [comment]
A marketing and positioning statement is not about capturing all possibilities. It is about saying something strong that you specialize. A hamburger at Mcdonald's is also tasty, but they sell it as "fast!".

**Comment by aartaka** [comment]
Fair, yes.
